misc/efi-stub/Makefile

```diff
@@ -0,0 +1,132 @@
+#
+# Copyright (c) 2011, Intel Corporation
```

Comment:
Is this a file that's coming from somewhere else and that you are copying as-is? I'm asking because I see the year in the header is 2011 (from 10 years ago).

Reply:
Thanks. I am thinking of adding readme to meta-acrn side, but I will add readme to under misc/efi-stub/. At least I will split it into two commits one for hv and another for efi-stub. The files derived from ACRN release_2.2. I will update the year in the headers. https://github.com/projectacrn/acrn-hypervisor/blob/release_2.2/misc/efi-stub/Makefile